OrbitClientProtos/capture_data.proto

```diff
@@ -41,7 +41,7 @@ message FunctionInfo {
     kOrbitTrackDoubleAsInt64 = 12;
   }
   OrbitType type = 10;
-  FunctionStats stats = 11;
+  reserved 11;
```

Comment:
nit: put it on the top - the first line of the message.

Reply:
Thanks, done!